PhysicsTools/PatAlgos/plugins/PATTriggerMatcher.cc

```diff
@@ -1,15 +1,47 @@
+// -*- C++ -*-
 //
+// Package:    PatAlgos
+// Class:      PATTriggerMatchSelector
 //
-#include "PhysicsTools/PatAlgos/plugins/PATTriggerMatchSelector.h"
-#include "CommonTools/UtilAlgos/interface/PhysObjectMatcher.h"
+/**
+  \class    pat::PATTriggerMatchSelector PATTriggerMatchSelector.h "PhysicsTools/PatAlgos/plugins/PATTriggerMatchSelector.h"
```

Comment:
Since the file name has changed, maybe also those comments should

Reply:
Okay